<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_waiter</class>
 <widget class="QDialog" name="Add_waiter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>941</width>
     <height>681</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>ADD WAITER</string>
   </property>
   <widget class="QPushButton" name="add_pushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>601</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:        </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="add_waiter_name_lineEdit">
         <property name="text">
          <string>  </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Surname:  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="add_waiter_surname_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Username: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="add_waiter_username_lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Password: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="add_waiter_pass_lineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="back_pushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>500</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
